HaveIBeenPwned/HaveIBeenPwned.csproj

```diff
@@ -55,7 +57,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="KeePass, Version=2.34.0.15991, Culture=neutral, PublicKeyToken=fed2ed7716aecf5c, processorArchitecture=MSIL">
-      <HintPath>$(USERPROFILE)\Dropbox\KeePass\KeePass.exe</HintPath>
+      <HintPath>$(MSBuildProgramFiles32)\KeePass Password Safe 2\KeePass.exe</HintPath>
```

Comment:
Is changing the hint path intended? It seems like Dropbox would be more common, but I don't know if there's some standard around this.

Reply:
As not all developers might use dropbox I changed the HintPath to the KeePass.exe in default installation folder.